how-to/index.md

```diff
@@ -35,6 +35,6 @@ If you have useful information that isn't listed in this page, please send a pul
 
   * [How to use Sudachi dictionary for synonym expansion](sudachi-dictionary.html)
 
-## ...
+## How to use with Spring/Hibernate
 
-  * ...
+  * [How to use with Spring/Hibernate](spring-hibernate.html)
```

Comment:
Could you add a new section instead of replacing the template?

```markdown
## How to use with Spring/Hibernate

  * [How to use with Spring/Hibernate](spring-hibernate.html)

## ...

  * ...
```

Reply:
Done